tester.go

```diff
@@ -97,6 +110,13 @@ func (t *T) Test(invoker Invoker) error {
 
 			msgType := msg["type"].(string)
 
+			if t.SkipMessageType(msgType) {
+				events.Log("Skip message for fixture %{fixture}v", fixture)
```

Comment:
Is there a way to print skipped tests in the report as well?

Reply:
yes, it's already been done. SEe the sample result in the comment above.

Reply:
ah perfect!